src/libs/actions/Wallet.ts

```diff
@@ -0,0 +1,364 @@
+            },
+        },
+    ];
+
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_ONFIDO,
+            value: {
+                isLoading: false,
+                errors: null,
+            },
+        },
+    ];
+
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_ONFIDO,
+            value: {
+                isLoading: false,
+                hasAcceptedPrivacyPolicy: false,
+            },
+        },
+    ];
+
+    const requestParams: IdentityVerification = {
+        onfidoData,
+    };
+
+    API.write('VerifyIdentity', requestParams, {
+        optimisticData,
+        successData,
+        failureData,
+    });
+}
+
+type WalletTerms = {
+    hasAcceptedTerms: boolean;
+    chatReportID: number;
+};
+
+/**
+ * Complete the "Accept Terms" step of the wallet activation flow.
+ *
+ * @param parameters.chatReportID When accepting the terms of wallet to pay an IOU, indicates the parent chat ID of the IOU
+ */
+function acceptWalletTerms(parameters: WalletTerms) {
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_TERMS,
+            value: {
+                isLoading: true,
+            },
+        },
+    ];
+
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_TERMS,
+            value: {
+                errors: null,
+                isLoading: false,
+            },
+        },
+    ];
+
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.USER_WALLET,
+            value: {
+                isPendingOnfidoResult: null,
+                shouldShowFailedKYC: true,
+            },
+        },
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_TERMS,
+            value: {
+                isLoading: false,
+            },
+        },
+    ];
+
+    type WalletTermsParameters = {
+        hasAcceptedTerms: boolean;
+        reportID: number;
+    };
+
+    const requestParams: WalletTermsParameters = {hasAcceptedTerms: parameters.hasAcceptedTerms, reportID: parameters.chatReportID};
+
+    API.write('AcceptWalletTerms', requestParams, {optimisticData, successData, failureData});
+}
+
+/**
+ * Fetches data when the user opens the InitialSettingsPage
+ */
+function openInitialSettingsPage() {
+    API.read('OpenInitialSettingsPage', {});
+}
+
+/**
+ * Fetches data when the user opens the EnablePaymentsPage
+ */
+function openEnablePaymentsPage() {
+    API.read('OpenEnablePaymentsPage', {});
+}
+
+function updateCurrentStep(currentStep: ValueOf<typeof CONST.WALLET.STEP>) {
+    Onyx.merge(ONYXKEYS.USER_WALLET, {currentStep});
+}
+
+function answerQuestionsForWallet(answers: unknown[], idNumber: string) {
```

Comment:
```suggestion
function answerQuestionsForWallet(answers: WalletQuestion[], idNumber: string) {
```

Looking [at this file](https://github.com/Expensify/App/blob/main/src/pages/EnablePayments/IdologyQuestions.js#L108), looks like `answers` type structure is something like this:

```ts
type WalletQuestion = {
    question: string;
    answer: string;
};
```